index.html

```diff
@@ -23,25 +23,160 @@
     </header>
 
 
-    <section class="container">
+    <section id="about">
 
-    <h1 class="primary-text">
+    <h1 class="title">
             I’m Miguel Cuello.
             Glad to see you!
-        </h1>
-        <p class="first-text">
+    </h1>
+        <p class="text">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
-        <h3>LET’S CONNECT</h3>
-        <ul id="social">
-            <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
-            <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
-            <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
-            <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+
+    <p class="project">LET’S CONNECT</p>
+
+    <ul class="social">
+        <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
+        <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
+         <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+        <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
+        <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
+    </ul>
+
+    </section>
+
+    <section  class="card" id="tonic">
+
+        <img src="images/portfolio.png" alt="a_screentshot" id="portfolio">
+
+        <h2 class="title tonic">Tonic</h2>
+
+        <img src="images/fame2.png" alt="frame" id="frame">
+
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
```

Comment:
- [ ] your design is currently inconsistent with the figma template design for how the technologies for each card should be displayed, the font color for the technologies are different please see the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=dBl8ND0TXYsfocKd-0) and apply the appropriate color for each technology :+1: 
- figma design
 ![fw-f](https://user-images.githubusercontent.com/89055502/235586964-534318cd-33ad-4cd0-ab9a-2da88172421f.png)
- your design
![fw-s](https://user-images.githubusercontent.com/89055502/235586993-05be1f73-1bad-4140-b003-a04663270dcc.png)
- [ ] please make this change in every single card :+1: